lib/init.ts

```diff
@@ -65,9 +50,6 @@ export function init(options: InitParams) {
 You can visit https://algolia.com/events/debugger instead.`);
   }
 
-  this._apiKey = options.apiKey;
-  this._appId = options.appId;
-
   setOptions(this, options, {
```

Comment:
I guess we can update this not to specify the defaults now, since they're assigned in the constructor, no? `setOptions` should only change what was passed.

Reply:
I'm not sure to understand what you mean, should we keep these two lines?

Reply:
currently, if you call `init` without `partial`, it will set the default value for the keys you didn't specify, if those are already set by something else, we probably want to reset them to the default if partial isn't given to preserve the previous behaviour, right? 

in v3 we could make partial the default and not an option, and then we don't need this anymore though

Reply:
Yes that's correct.